crates/hir-ty/src/tests/simple.rs

```diff
@@ -3692,39 +3692,6 @@ fn main() {
     );
 }
 
-#[test]
-fn infer_bad_lang_item() {
```

Comment:
why the removed test? seems like we still should at least not crash on this

Reply:
It is a bit difficult (although not impossible) to support this due to not using `TyBuilder`, and this doesn't seem particularly important use case to support.